README.md

```diff
@@ -47,34 +47,35 @@ Schemasheets is part of the LinkML toolset that allows a LinkML data description
 Generation of spreadsheet through linkml schemasheets
 
 ```
-linkml2schemasheets-template -i nmr.yaml -o nmr_concise.tsv -s concise
+linkml2schemasheets-template -i src/nmr_schema/nmr_schema.yaml -o nmr_concise.tsv -s concise
 ```
 or
 ```
-linkml2schemasheets-template -i nmr.yaml -o nmr_exhaustive.tsv -s exhaustive
+linkml2schemasheets-template -i src/nmr_schema/nmr_schema.yaml -o nmr_exhaustive.tsv -s exhaustive
 ```
 
 ### 2. Generate json schema
 
 Generating a corresponding (closed world) JSON-Schema definition (`nmr.schema.json`) for the NMR datasets:
 
 ```
-gen-json-schema --closed nmr.yaml  >nmr.schema.json
+gen-json-schema --closed src/nmr_schema/nmr_schema.yaml  >nmr.schema.json
 ```
 
 ### 3. Simple worked examples to validate the csv input metadata
 
 Check ("valdate") that some csv metadata (`my-nmr-metadata.csv`) complies with the SFX metadata defintion (`nmr.yaml`):
 
 ```
-linkml-validate -s nmr.yaml my-nmr-metadata.csv
+linkml-validate -s src/nmr_schema/nmr_schema.yaml src/example_data/my-nmr-metadata.csv
```

Comment:
This fails. I think it's because the `my-nmr-metadata.csv`, I don't have the csv file.

Reply:
I renamed the file, correcrted